RELEASING.md

```diff
@@ -0,0 +1,138 @@
+<!-- markdownlint-configure-file { "MD013": { "line_length": 120 } } -->
+# Releasing
+
+## Overview
+
+This repo contains three types of release artifacts:
+
+* **Go Binaries** for Terraform Engine, Policy Generator, and Terraform Importer.
+* **Templates** for the Terraform Engine ("templates" for short).
+* **Policy templates** for the Policy Generator ("policies" for short).
+
+These three groups of artifacts are not coupled with each other, so they are **released separately**.
+
+### Versioning
+
+Releases are tagged with [Git Annotated Tags](https://git-scm.com/book/en/v2/Git-Basics-Tagging) following [semantic
+versioning](https://semver.org/) (semver):
+
+* `v1.x.x` - MAJOR version, increments on breaking changes.
+* `v1.2.x` - MINOR version, increments on backwards-compatible new features.
+* `v1.2.3` - PATCH version, increments on backwards-compatible bug fixes.
+
+The only exception to the above is the `0.x.x` version. As described in semver, this version is for initial development;
+anything may change at any time, and the public API should not be considered stable.
+
+We have the following release tags:
+
+* Ex. `v0.1.2` for binaries
+* Ex. `templates-v0.1.2` for templates.
```

Comment:
Are we versioning all templates as a single unit or will each template have it's own version history? I think we're versioning all templates as a single block, so any single update would revision all templates, but can we clarify that here?

Reply:
All templates are versioned as a single unit, same for policies. I'll try to clarify it.

Reply:
Done